### PR TITLE
Modify release schedules to add cherry-pick deadlines for July '21

### PR DESCRIPTION
**Pull request by puerco**
This PR updates the patch releases page markdown and the automatic schedule to add the cherry-pick deadlines for July '21 and also to add the upcoming releases in the machine-readable yaml schedule.

I made a slight modification to the table in the general calendar to add the CP dates there too to make it easier to see them at a glance.

### CODE REVIEW

**Comment by puerco** [comment]
Mmh this is odd, I think I'm in the necessary OWNERS aliases and the PR should have gotten approved. Funny.

/assign @jimangel

**Review by cpanato — APPROVED** [review]
thanks for updating this!
/lgtm

**Review by cpanato — COMMENTED** [review]
/hold for any other comments/review

**Comment by cpanato** [comment]
> Mmh this is odd, I think I'm in the necessary OWNERS aliases and the PR should have gotten approved. Funny.
> 
> /assign @jimangel

you are in the file, but the way this repo is configured in prow it does not approve the PR by default if you are an approver

**Comment by cpanato** [comment]
looks like no objections

/hold cancel

**Pull request merged**
